README.md

```diff
@@ -35,3 +35,38 @@ If using without Pants, run `flake8 your_module.py` [as usual](http://flake8.pyc
 ## Migration from `pantsbuild.pants.contrib.python.checks.checker`
 
 TODO
+
+## Development
+
+We use [Poetry](https://python-poetry.org) for dependency management, building the wheel, and publishing.
+
+### To install
+
+```bash
+$ poetry install
+```
+
+This will use whichever version `python` points to. You can set `poetry env use python3.7`, for example, to point to a different Python discoverable on your PATH.
+
+### To test
+
+We use [tox](https://testrun.org/tox/en/latest/) for test automation. To run the test suite, invoke tox:
+
+```bash
+$ tox
+```
+
+You may also manually test by building a PEX with this plugin, as follows:
+
+```bash
+$ pex flake8 . --entry-point flake8 --output-file flake8.pex
+$ vim example.py  # add whatever content you want
+$ ./flake.pex example.py
+```
+
+### To publish
+
+```bash
+$ poetry build
```

Comment:
Presumably `poetry publish` is enough.

Reply:
Negative, but `poetry --build publish` would work. https://python-poetry.org/docs/libraries/#publishing-to-pypi